index.js

```diff
@@ -120,9 +137,25 @@ async function print(argv) {
     await page.goto(url, buildNavigationOptions(argv));
 
     console.error(`Writing ${argv.output || 'STDOUT'}`);
+
+    let height, width;
+    if (argv.magicformat) {
+        height = (await page.evaluate(
+            'Math.max(document.body.scrollHeight, document.body.offsetHeight, document.documentElement.clientHeight, document.documentElement.scrollHeight, document.documentElement.offsetHeight)'));
+        width = '1366';
+        argv.format = undefined;
+    }
+
+    if (argv.injectjs) {
+        await page.evaluate(`(async () => {${fs.readFileSync(argv.injectjs)}})()`);
+    }
+
+    if (argv.emulatemedia) await page.emulateMedia(argv.emulatemedia);
```

Comment:
linter conflict here, please break out the inlining

```suggestion
    if (argv.emulatemedia) {
        await page.emulateMedia(argv.emulatemedia);
    }
```

Reply:
done